app/src/test/kotlin/com/softteco/template/ui/feature/resetPassword/ResetPasswordViewModelTest.kt

```diff
@@ -0,0 +1,170 @@
+package com.softteco.template.ui.feature.resetPassword
+
+import androidx.lifecycle.SavedStateHandle
+import app.cash.turbine.test
+import com.softteco.template.BaseTest
+import com.softteco.template.data.base.error.Result
+import com.softteco.template.data.profile.ProfileRepository
+import com.softteco.template.data.profile.dto.NewPasswordDto
+import com.softteco.template.navigation.AppNavHost
+import com.softteco.template.ui.feature.PasswordFieldState
+import com.softteco.template.utils.MainDispatcherExtension
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.types.shouldBeTypeOf
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.impl.annotations.RelaxedMockK
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import kotlin.time.Duration.Companion.seconds
+
+@ExtendWith(MainDispatcherExtension::class)
+class ResetPasswordViewModelTest : BaseTest() {
+
+    @RelaxedMockK
+    private lateinit var repository: ProfileRepository
+    private lateinit var viewModel: ResetPasswordViewModel
+
+    @Test
+    fun `when valid password and reset password button is enabled then success state is emitted`() =
+        runTest {
+            val token = "testToken"
```

Comment:
Move strings to constants. 
like: 
`private const val TOKEN = "token"`
Right here at the top of this file.

Reply:
to declare variables as `const val` I need to create a companion object. Why I can't declare it as `private val token = "token"` at the top of this file?

Reply:
You don't need  a companion object for that